src/main/java/org/jabref/logic/bibtexkeypattern/BracketedPattern.java

```diff
@@ -570,6 +570,32 @@ public static String firstAuthor(String authorField) {
         if (authorList.isEmpty()) {
             return "";
         }
+        if (authorList.getAuthor(0).getVon().toString().equals("Optional.empty")) {
```

Comment:
Much easier, Optionals have a method names isPresent https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html

Reply:
Thanks for your suggestion! We will change it better.